packages/react-core/src/demos/examples/Tabs/GrayTabs.tsx

```diff
@@ -0,0 +1,158 @@
+                  activeKey={activeNestedTabKey}
+                  hidden={10 !== activeNestedTabKey}
+                >
+                  <TabContentBody>
+                    {
+                      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce in odio porttitor, feugiat risus in, feugiat arcu. Nullam euismod enim eget fringilla condimentum. Maecenas tincidunt et metus id aliquet. Integer et fermentum purus. Nulla tempor velit arcu, vitae semper purus iaculis at. Sed malesuada auctor luctus. Pellentesque et leo urna. Aliquam vitae felis congue lacus mattis fringilla. Nullam et ultricies erat, sed dignissim elit. Cras mattis pulvinar aliquam. In ac est nulla. Pellentesque fermentum nibh ac sapien porta, ut congue orci aliquam. Sed nisl est, tempor eu pharetra eget, ullamcorper ut augue. Vestibulum eleifend libero eu nulla cursus lacinia.'
+                    }
+                  </TabContentBody>
+                </TabContent>
+                <TabContent
+                  key={11}
+                  eventKey={11}
+                  id={`tabContent${11}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={11 !== activeNestedTabKey}
+                >
+                  <TabContentBody>Control plane panel</TabContentBody>
+                </TabContent>
+                <TabContent
+                  key={12}
+                  eventKey={12}
+                  id={`tabContent${12}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={12 !== activeNestedTabKey}
+                >
+                  <TabContentBody>Operators panel</TabContentBody>
+                </TabContent>
+                <TabContent
+                  key={13}
+                  eventKey={13}
+                  id={`tabContent${13}`}
+                  activeKey={activeNestedTabKey}
+                  hidden={13 !== activeNestedTabKey}
+                >
+                  <TabContentBody>Virtualization panel</TabContentBody>
+                </TabContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </GridItem>
+      <GridItem xl={4} md={6}>
+        <Flex direction={{ default: 'column' }} className="pf-u-h-100">
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <Card isFullHeight>
+              <CardHeader>
+                <Title headingLevel="h3">Title of Card</Title>
+              </CardHeader>
+            </Card>
+          </FlexItem>
+          <FlexItem flex={{ default: 'flex_1' }}>
+            <Card isFullHeight>
+              <CardHeader>
+                <Title headingLevel="h3">Title of Card</Title>
+              </CardHeader>
+            </Card>
+          </FlexItem>
+        </Flex>
+      </GridItem>
+    </Grid>
+  );
+
+  return (
+    <DashboardWrapper hasNoBreadcrumb>
+      <PageSection isWidthLimited variant={PageSectionVariants.light}>
+        <Title headingLevel="h1" size="2xl">
```

Comment:
In the core demos, we're forcing a lighter font-weight on these titles. I'm in favor of what you have here, since our titles and card titles are bold by default. Seems like we would just follow that general pattern for consistency so as not to encourage inconsistency unless there's a specific reason for it. wdyt @mcarrano @mmenestr?

Reply:
I don't see any reason to not just stick with our default styles here. @mceledonia any thoughts?

Reply:
@gitdallas confirmed with @mceledonia that the way you've done it is good - we'll update the core demos.